relay/sources/relayd/tests/files/runlogs/2020-03-24T12:29:54+00:00@root.json

```diff
@@ -5,6 +5,19 @@
     },
     "config_id": "20200324-102421-f6947255",
     "reports": [
+        {
```

Comment:
the start was skipped due to the non-timestamped line in https://github.com/Normation/rudder/blob/5fcf5c1f2ad9b5c255ce59881284b7b2b8115a3f/relay/sources/relayd/tests/files/runlogs/2020-03-24T12:29:54%2B00:00%40root.log#L1. They should never appear there is the wild, but I kept them to ensure the parser does not explode if it happens.